components/script/canvas_context.rs

```diff
@@ -39,6 +39,8 @@ pub(crate) trait CanvasContext {
 
     fn resize(&self);
 
+    fn reset_bitmap(&self);
```

Comment:
It might be nice to have a little documentation here that explains what this interface should do, both to make it easier for people implementing this for WebGL and WebGPU and also for people reading the code.

Reply:
Done